sdk/typescript/introspector/decorators/decorators.ts

```diff
@@ -6,3 +6,4 @@ import { registry } from "../registry/registry.js"
 export const object = registry.object
 export const func = registry.func
 export const field = registry.field
+export const daggerEnum = registry.daggerEnum
```

Comment:
I don't really like repeating the package name in a var like this. For example (hypothetically), if the naming clash with `enum` was just shadowing a built-in instead of a syntax error, you could resolve the conflict with:

```diff
- import { enum } from '@dagger.io/dagger';
+ import { enum as enumType } from '@dagger.io/dagger';
```

I've resolved the conflict in Python with `@object_type` and `@enum_type` because there's a built-in `object` and `enum` in the std lib. Adding "type" matches GraphQL since they both create different types. Would be the same if we decided to support custom _input types_ (there's also a built-in `input` in Python).

So how about `enumType` instead?

Reply:
I like that, I can make the switch